packages/dma-contracts/tasks/common/verification-utils.ts

```diff
@@ -100,3 +115,90 @@ export class ActionsDatabase {
     })
   }
 }
+
+export function validateActionHashes(
+  operationHashes: string[],
+  operationOptionals: boolean[],
```

Comment:
maybe `optionalOperationActions` ?

Reply:
or `operationOptionalActions`

Reply:
I've gone with `operationIsActionOptional` because for me the other names imply that it is a list of optional actions, when in reality is a list of whether the corresponding action is optional. Let me know if you like it!